device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-C128/buffers_defaults_t2.j2

```diff
@@ -0,0 +1,45 @@
+
+{#
+    Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES.
+    Apache-2.0
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+#}
+{% set default_cable = '1500m' %}
+{%-set ports2cable = {
+            'spinerouter_leafrouter' : '1500m',
+            'spinerouter_spinerouter' : '1500m'
+       }
+-%}
+{% set ingress_lossless_pool_size =  '36405216' %}
+{% set ingress_lossless_pool_xoff  =  '11567088' %}
+{% set egress_lossless_pool_size =  '60817392' %}
+{% set egress_lossy_pool_size =  '36405216' %}
```

Comment:
As a convention, we round down by cell size for egress_lossless_pool only. For the rest sizes, we do not round them down.
```suggestion
{% set ingress_lossless_pool_size =  '36405248' %}
{% set ingress_lossless_pool_xoff  =  '11567104' %}
{% set egress_lossless_pool_size =  '60817392' %}
{% set egress_lossy_pool_size =  '36405248' %}
```